src/atom-config/job-config.js

```diff
@@ -0,0 +1,20 @@
+'use babel'
+
+// This is parsed in main script *before* idleCallbacks,
```

Comment:
I haven't been benchmarking any of this since this branch isn't anywhere close to merging into `master`, but be warned that every import adds ~2 ms to the time, of which we have 5 ms total before users start seeing warnings about us.

Reply:
Hmm... that sounds pretty restrictive. I think I'll worry about that later. Aim for a more ideal setup first, and then optimize where required. But that does mean that my comment there is useless, because once I start trying to optimize this will probably need to be replaced with a custom pick function anyway instead of importing from ramda at all.